docs/servo_power_test/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "cr50",
```

Comment:
could you update the name to match with each project? (Also you can update "author" attribute as well if you want ;) )